src/main/java/org/fogbowcloud/manager/core/BehaviorPluginsHolder.java

```diff
@@ -1,37 +1,44 @@
 package org.fogbowcloud.manager.core;
 
+import org.fogbowcloud.manager.core.models.tokens.TokenGeneratorPlugin;
 import org.fogbowcloud.manager.core.plugins.behavior.authorization.AuthorizationPlugin;
 import org.fogbowcloud.manager.core.plugins.behavior.authentication.AuthenticationPlugin;
 import org.fogbowcloud.manager.core.plugins.behavior.identity.FederationIdentityPlugin;
 import org.fogbowcloud.manager.core.plugins.behavior.mapper.FederationToLocalMapperPlugin;
 
 public class BehaviorPluginsHolder {
 
-    private AuthorizationPlugin authorizationPlugin;
+    private TokenGeneratorPlugin tokenGeneratorPlugin;
```

Comment:
Out of curiosity, the token generator plugin is going to be described as a behaviour plugin in the paper?